### DISCUSSION TITLE
504 error after adding site

### DISCUSSION THREAD

**Discussion by sc00zy** [akeeba/panopticon#539] [Q&A (Community Support)]
I have quite a few sites (30 or so) now in my Panopticon installation (1.1.0) and tonight I was adding a couple of websites more. But all of the sudden after  trying to add a new site I get a 504 error and I can't login to Panopticon anymore at all. If I try I keep getting the 504 error.

In the error log I see lines like this:

mod_fcgid: read data timeout in 60 seconds
End of script output before headers: index.php
mod_fcgid: read data timeout in 60 seconds, referer: https://panopticon.mydomain.nl/
End of script output before headers: index.php, referer: https://panopticon.mydomain.nl/
6004#0: *2993434 upstream timed out (110: Connection timed out) while reading response header from upstream

I'm not sure what I can do right now to fix it. Thanks in advance for thinking with me.

**Comment by nikosdion** [comment]
See https://github.com/akeeba/panopticon/issues/537#issuecomment-1905961948

**Reply by sc00zy** [reply]
Thanks. I'm not sure how to install this on top of Panopticon installation. Just overwriting all files will give an error. Can you please guide me through it a bit? Thanks!

**Reply by nikosdion** [reply]
You would need to run `composer install` as noted in https://github.com/akeeba/panopticon/wiki/Install-from-Git  If this is too technical for you, just wait for next week's release of version 1.1.1.

**Reply by sc00zy** [reply]
Thank you. I will wait, but I had to disable Panopticon because my server was having a bad time with over 150 MySQL jobs 😬
